.ci/generate_bear_requirements.py

```diff
@@ -46,13 +46,15 @@
     'PipRequirement',
     'NpmRequirement',
     'GemRequirement',
+    'ComposerRequirement("php_codesniffer")',
```

Comment:
no that's not what I meant^^ This one should be just `ComposerRequirement`. You shall modify the actual bear itself that is using `ComposerRequirement("php_codesniffer")`! :)

Reply:
You meant to say that i should change [this](https://github.com/coala/coala-bears/blob/master/bears/php/PHPCodeSnifferBear.py#L29) line.